package.json

```diff
@@ -51,6 +51,7 @@
     "karma-mocha": "^1.3.0",
     "karma-mocha-reporter": "^2.2.5",
     "karma-opera-launcher": "^1.0.0",
+    "karma-respec-github-action-reporter": "github:sidvishnoi/karma-respec-github-action-reporter",
```

Comment:
If this is mainly for ReSpec maybe make it a local module?

Reply:
Karma only picks from node_modules :disappointed:

Reply:
Seems there is a way to `require()` a module though: https://karma-runner.github.io/4.0/config/plugins.html

Reply:
Ouch. Didn't see it.

Reply:
Make sure that you also add `"karma-*"` to the `plugins` field or it will stop autoloading karma plugins.